pid_gains:
  vel_Kp: [20.0, 20.0, 20.0, 8.0, 8.0, 8.0]
  vel_Kd: [0.025, 0.025, 0.025, 0.15, 0.15, 0.15]

  position_Kp: [80.5, 80.5, 80.5]
  position_Ki: [0.005, 0.005, 0.005]
  position_Kd: [40.0, 40.0, 40.0]
  att_Kp: [12.0, 12.0, 20.0]
  